.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request. The three fields below are mandatory. -->
+
+## Summary
+
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
+
+## Changelog
+
+<!-- Help reviewers and the release process by writing your own changelog entry. For an example, see:
+https://reactnative.dev/contributing/changelogs-in-pull-requests -->
+
+[CATEGORY] [TYPE] - Message
+
+## Test Plan
+
+<!-- Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes the user interface. -->
```

Comment:
So nice field!! 👍 With this test plan, contributors will run build and test when they create PR! And, it also helps for reviewer to review PRs. 💯